felix/dataplane/linux/int_dataplane.go

```diff
@@ -958,6 +959,12 @@ func ConfigureDefaultMTUs(hostMTU int, c *Config) {
 	if c.VXLANMTU == 0 {
 		log.Debug("Defaulting VXLAN MTU based on host")
 		c.VXLANMTU = hostMTU - vxlanMTUOverhead
+
+		if c.IPv6Enabled {
+			log.Debug("IPv6 is enabled, defaulting VXLAN MTU based on host")
```

Comment:
Hm,  I suppose we'll hit this even if VXLAN is only being used for IPv4 pools? 

e.g., VXLAN for IPv4, unencap'd for v6?

Reply:
Maybe we can do a follow-on to fine tune this based on whether there are any IPv6 pools with VXLAN enabled, but if that's not trivial we can include it in a patch release later.

Reply:
Hm... That's correct, thanks for catching this... Yes, looking at the pools directly to make sure that VXLAN+IPv6 is being used would be best. I'll do that in a follow-on PR.